src/main/webapp/dataverse.xhtml

```diff
@@ -110,7 +110,8 @@
                                           data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['dataverse.name.title']}"></span>
                                 </h:outputLabel>
                                 <div class="form-col-container">
-                                    <p:inputText id="name" styleClass="form-control" value="#{DataversePage.dataverse.name}" required="true"/>
+                                    <p:inputText id="name" styleClass="form-control" value="#{DataversePage.dataverse.name}" />
```

Comment:
How was this not there before??

Reply:
These required attributes were adding in 4.18 for issue #5615 in an attempt to improve accessibility of the forms.

Reply:
I can take the approach of trying to get the required message from the ValidationMessages.properties file and see if that works.

Reply:
Looks like it was just a miss. No one leaves the name blank? or something.

Reply:
The attributes do not appear necessary for the validation to function, but it is in order to put an accessibility friendly `aria-required="true"` in the input component code, as the red asterisk visual indicator is not sufficient.

Reply:
@mheppler So do all the others ones need to have it added back?

Reply:
I believe so. I tried using `pt:aria-required="true"` in the p:input component to get the attribute in there another way, but it was just ignored.

Reply:
I returned all of the required='true' where applicable. I am now getting the requiredMessage from the Bundle because the ValidationMessages.properties are only used by the Bean Validations